storage/block/how-to/mount-and-use-volume.mdx

```diff
@@ -88,28 +88,28 @@ Once your file system is created, you need to mount it.
     mount -o defaults /dev/sda /mnt/block-volume
     ```
 
-  ```sh no-copy
-       **defaults**
-            Use  the  default  options: **rw**, **suid**, **dev**,
-            **exec**, **auto**, **nouser**, and **async**.
-  ```
+    ```sh no-copy
+        **defaults**
+              Use  the  default  options: **rw**, **suid**, **dev**,
+              **exec**, **auto**, **nouser**, and **async**.
+    ```
 
-  If you want to see all available options, you can run `man mount` on your instance.
+    If you want to see all available options, you can run `man mount` on your instance.
 3. Make sure your file system is properly mounted by running the `lsblk` command.
     ```
     lsblk
     ```
 
-  You should see an output like the following. Check the `MOUNTPOINT` field.
+    You should see an output like the following. Check the `MOUNTPOINT` field.
 
-  ```sh no-copy
-  root@scw-festive-agnesi:~# lsblk
-  NAME    MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
-  sda       8:0    0 23.3G  0 disk /mnt/block-volume
-  vda     252:0    0 18.6G  0 disk
-  ├─vda1  252:1    0 18.5G  0 part /
-  └─vda15 252:15   0  100M  0 part /boot/efi
-  ```
+    ```sh no-copy
+    root@scw-festive-agnesi:~# lsblk
+    NAME    MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
+    sda       8:0    0 23.3G  0 disk /mnt/block-volume
+    vda     252:0    0 18.6G  0 disk
+    ├─vda1  252:1    0 18.5G  0 part /
+    └─vda15 252:15   0  100M  0 part /boot/efi
+    ```
 
 ## How to use fstab for Persistent Mounting
```

Comment:
can't review line 118 for some reason, but there's another "instance" needing capitalization

Reply:
fixed! merci